jvm/ml4ir-inference/pom.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>ml4ir</groupId>
+    <artifactId>ml4ir-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>ml4ir-inference</artifactId>
+  <packaging>jar</packaging>
+
+  <build>
+    <sourceDirectory>src/main/scala</sourceDirectory>
+    <testSourceDirectory>src/test/scala</testSourceDirectory>
+
+    <plugins>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-site-plugin</artifactId>
+          <version>3.7.1</version>
+      </plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-project-info-reports-plugin</artifactId>
+          <version>3.0.0</version>
+      </plugin>
+      <!-- mixed scala/java compile -->
+      <plugin>
+	<groupId>net.alchim31.maven</groupId>
+	<artifactId>scala-maven-plugin</artifactId>
+        <version>3.3.1</version>
+	<executions>
+	  <execution>
+	    <id>compile</id>
+	    <goals>
+	      <goal>compile</goal>
+	    </goals>
+	    <phase>compile</phase>
+	  </execution>
+	  <execution>
+	    <id>test-compile</id>
+	    <goals>
+	      <goal>testCompile</goal>
+	    </goals>
+	    <phase>test-compile</phase>
+	  </execution>
+	  <execution>
+	    <phase>process-resources</phase>
+	    <goals>
+	      <goal>compile</goal>
+	    </goals>
+	  </execution>
+	</executions>
+      </plugin>
+      <plugin>
+	<artifactId>maven-compiler-plugin</artifactId>
+	<configuration>
+	  <source>1.7</source>
+	  <target>1.7</target>
+	</configuration>
+      </plugin>
+		<!-- disable surefire - - >
```

Comment:
Will we need this in the future? If not, then remove

Reply:
This is the way to make sure maven can work with scalatest: 
http://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin